src/libs/antares/study/parameters.cpp

```diff
@@ -1020,10 +962,33 @@ bool Parameters::loadFromINI(const IniFile& ini, uint version, const StudyLoadOp
     reset();
     // A temporary buffer, used for the values in lowercase
     String value;
-
+    String sectionName;
+    typedef bool (*Callback)(Parameters&, const String&, const String&, const String&, const String&, uint);
+
+    static std::map<String,Callback> sectionAssociatedToKeysProcess = {
+            {"general",&SGDIntLoadFamily_General},
+            {"input",&SGDIntLoadFamily_Input},
+            {"output",&SGDIntLoadFamily_Output},
+            {"optimization",&SGDIntLoadFamily_Optimization},
+            {"other preferences",&SGDIntLoadFamily_OtherPreferences},
+            {"advanced parameters",&SGDIntLoadFamily_AdvancedParameters},
+            {"playlist",&SGDIntLoadFamily_Playlist},
+            {"variables selection",&SGDIntLoadFamily_VariablesSelection},
+            {"seeds",&SGDIntLoadFamily_Seeds}};
+
+    Callback* handleAllKeysInSection;
```

Comment:
`Callback` is already a function pointer. This is equivalent to 
```cpp
function_t** Callback;
```
I suggest you remove one indirection level.

Reply:
Ok. Commits 96a1f00 (correction of 0a225bf).